### DISCUSSION TITLE
6.2.0

### DISCUSSION THREAD

**Discussion by graphefruit** [graphefruit/Beanconqueror#413] [General]
Pressure devices
Support of 'Do it yourself' Popsicle - Sensor - Thanks to Mike and Shay
Autostart timer on a specific pressure - editable in settings
Press of pressure gauge, will reset pressure to 0
Display name of pressure device and mac-address on connect

Bluetooth scales
Support of Eureka Precisa - Thanks to Mimoja
Define the behavior of the bluetooth scale on a new brew - editable in settings
Define which graphs should be displayed on filter or espresso - editable in settings
Sequential processing of bluetooth commands to reduce issues
Display name of bluetooth scale and mac-address on connect

Brews
Supports now milliseconds - Activate in settings
Filtersystem expanded, search for roast type and roastery
You can reconnect your bluetooth scale or pressure device on the brew add/edit page

Beans
Share your bean (without server needed) - Thanks to Mike
Show all brews to your bean
Support bean rating like for brews - editable in settings

More languages
Support of Turkish - Thanks to Halil Portakal
Support of Chinese - Thanks to Jiageng

Mills
Display of the last used grind setting, aswell as the last used bean

Improvements
Error message when a bean is scanned, which has not been approved yet
The edit of the brew-date is not activated by default

Others
Typos fixed
Fixed an issue that you didn't see the star-rating even with normal settings
Older brew graphics can now be exported again
Update to Ionic 6.3.0
Smaller bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/graphefruit/Beanconqueror/releases/tag/v.6.2.0'>6.2.0</a>.</em>

**Comment by frdmn** [comment]
Is there any information available about the Popsicle sensor? 

Other than this mention in [this Kaffee-Netz thread](https://www.kaffee-netz.de/threads/beanconqueror-app.111249/page-52#post-2153707) I haven't been able to figure out what this is exactly and how to obtain/build it.

**Reply by graphefruit** [reply]
Actually this topic is still in development.
I'm having contact some times with the creators, it will be published when finished, but its also a hobby project, so still some time to go I think.
I'm not included in the development or something like this.
You can see the popsicle sensor in this video: https://www.youtube.com/watch?v=nO6wihXknPk&ab_channel=jkimmakes
